config/rbac/role_binding.yaml

```diff
@@ -14,6 +14,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
   name: che-operator
+  namespace: eclipse-che
```

Comment:
Is there a reason to start including the namespace in these resources?

Also I'm not clear whether the leader-election role/rolebindings are required for this PR or if they're just updates.

Reply:
1. No reason, I just wanted to explicitly specify namespace into resources.

2. Initially DWO roles were included into Che Operator ones [1]. Since I removed them, I had to add missing ones.

[1] https://github.com/eclipse-che/che-operator/pull/1520/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L376-L423